Bottom: dd8d933f069e480214d8c94c47c8e63a86f99fa3
Top:    03431210b87a1abae083762d8f4019492b91d0da
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-13 08:39:31 +0800

Refresh of vhost_net-passing-raw-xdp-buff

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 8572960..144b549 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -570,6 +570,7 @@ static void handle_tx_copy(struct vhost_net *net)
 {
 	struct vhost_net_virtqueue *nvq = &net->vqs[VHOST_NET_VQ_TX];
 	struct vhost_virtqueue *vq = &nvq->vq;
+	struct xdp_buff xdp;
 	unsigned out, in;
 	int head;
 	struct msghdr msg = {
@@ -675,7 +676,6 @@ static void handle_tx_zerocopy(struct vhost_net *net)
 {
 	struct vhost_net_virtqueue *nvq = &net->vqs[VHOST_NET_VQ_TX];
 	struct vhost_virtqueue *vq = &nvq->vq;
-	struct xdp_buff xdp;
 	unsigned out, in;
 	int head;
 	struct msghdr msg = {
